When building for Qt5, the python extension needs QtWidgets and QtPrintSupport
because things moved there from QtGui in Qt4.  Sent upstream.

Index: qscintilla2-2.8.3+dfsg/Python/configure.py
===================================================================
--- qscintilla2-2.8.3+dfsg.orig/Python/configure.py	2014-07-05 10:02:57.882832733 -0400
+++ qscintilla2-2.8.3+dfsg/Python/configure.py	2014-07-05 10:02:57.874832733 -0400
@@ -1423,6 +1423,9 @@
     if includepath:
         pro.write('INCLUDEPATH += %s\n' % includepath)
 
+    if target_config.pyqt_package == 'PyQt5':
+        pro.write('INCLUDEPATH += %s\n' % (target_config.qt_inc_dir + '/QtWidgets ' + target_config.qt_inc_dir + '/QtPrintSupport'))
+
     # Make sure the SIP include directory is searched before the Python include
     # directory if they are different.
     pro.write('INCLUDEPATH += %s\n' % quote(target_config.sip_inc_dir))
